.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,46 @@
+name: Integration tests
+
+on:
+  push:
+    branches:
+      - "master"
+      - "release*"
+
+permissions:
+  contents: read
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout latest commit in the PR
+        uses: actions/checkout@v3
+      - name: Set up Docker QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: "1.18"
```

Comment:
nit: 1.19?

Reply:
yeah, 1.19 seems a good fit for the workflows, i'll update it.

Reply:
addressed :D